index.js

```diff
@@ -1,25 +1,28 @@
 import { DateTime } from 'luxon';
-const listBooks = document.querySelector('.list-books');
+import Book from './modules/index.js';
+
 const form = document.querySelector('.form-input');
 const [title, author] = form.elements;
 const [navList, navAdd, navContact] = document.querySelectorAll('.list-item');
 const allBooks = document.querySelector('.all-books');
 const addBook = document.querySelector('.add-book');
 const contact = document.querySelector('.contact');
+const divTime = document.querySelector('.date-time');
+
+function setDate() {
+  const dt = DateTime.now();
+  divTime.textContent = dt.toLocaleString(DateTime.DATETIME_FULL_WITH_SECONDS);
+}
```

Comment:
- [ ] As one of the requiremments says  that `you should use arrow syntax for functions.`  Good job I can all your functions use the arrow syntax exept this :point_up: . let's kindly change this syntax to an arrow